[IMP] MRP: updated bills of materials rst content (#2611)
odoo/documentation: content/applications/inventory_and_mrp/manufacturing/management/product_variants.rst

=============================

> Managing product variant BoMs

Reviewer: Suggestion:
```suggestion
Managing product variant Bill of Materials (BoM)
```

=============================

> In Odoo, one bill of material can be used for multiple variants of the same product. Having a

Reviewer: Checking for consistency on `bill of material` in line 5.

Everywhere else in this documentation Bill of Materials is capitalized and abbreviated. Materials is also plural

```suggestion
In Odoo, one Bill of Materials (BoM) can be used for multiple variants of the same product. Having
```

consolidated BoM for a product with variants saves time by preventing the need to manage multiple

> BoMs. To activate variants, simply navigate to :menuselection:`Inventory --> Configuration --> Settings --> Variants`. For more information on configuring product variants, refer to this page. 

Reviewer: Lines 6-7:

- missing `:abbr:` tags
- suggestion: new paragraph at "To activate..." since this goes into instruction and the sentences before this were introductory/contextual.
- line break needed at or before 100th char on line 7
- "For more information..." ideally is a note with a hyperlink to product variants doc.
```suggestion
a consolidated :abbr:`BoM (Bill of Materials)` for a product with variants saves time by preventing
the need to manage multiple :abbr:`BoMs (Bills of Materials)`.

To activate variants, simply navigate to :menuselection:`Inventory --> Configuration --> Settings
--> Variants`. 

.. note:: 
   For more information on configuring product variants, refer to [doc article name + link].

> .. image:: product_variants/bom-variants1.png
>    :align: center
>    :alt: 

Reviewer: - image filename needs to be updated
- missing `:alt:` text

```suggestion
.. image:: product_variants/enable-product-variants-setting.png
   :align: center
   :alt: Enable the Product Variants setting in Odoo in order to create attachable BoMs.
```

Once product variants are fully configured, components may be assigned to specific variants on the
bill of materials. Each component may be assigned to multiple variants. Components with no variant
specificed are used in every variant of the product.

When defining BoMs for product variants by component assignment, the :guilabel:`Product Variant`

> field in the main section of the BoM should be left blank, as shown below. This field is used only

Reviewer: Remove dependency on the image in the text.
```suggestion
field in the main section of the BoM should be left blank. This field is used only
```

when creating a BoM for one specific product variant.

.. image:: product_variants/kit-bom1.png

>    :align: center

Reviewer: - image filename needs to be updated
- missing :alt: text

> Note that the same principle applies for the configuration 
> of operations.

Reviewer: no line break needed here.
```suggestion
Note that the same principle applies for the configuration of operations.
```

.. image:: product_variants/kit-bom2.png

>    :align: center

Reviewer: - image filename needs to be updated
- missing :alt: text

> And for the production of by-products. 

Reviewer: Lines 24/25 and line 30 are fragments of the same sentence.

Ideally these get combined into one sentence OR more/complete context is added to both paragraphs in order to strengthen the written documentation and + break the dependency on the images to explain what's going on.

.. image:: product_variants/kit-bom3.png

>    :align: center

Reviewer: - image filename needs to be updated
- missing :alt: text
